src/lib/addons/slack-app.ts

```diff
@@ -70,7 +69,6 @@ export default class SlackAppAddon extends Addon {
                     event,
                     noAccessTokenMessage,
                 );
-                return;
```

Comment:
Should we keep this return? It stops executing the rest of the code when there's no access token